<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>476</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b_seeothers">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZOBACZ INNE POMIARY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tw√≥j puls:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZAPISZ POMIAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_puls">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>120</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>BPM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_start">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ROZPOCZNIJ POMIAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="widget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>165</width>
     <height>168</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
